crates/hotshot-state-prover/src/circuit.rs

```diff
@@ -116,236 +114,191 @@ impl AsRef<[Variable]> for LightClientStateVar {
     }
 }
 
-#[derive(Clone, Debug)]
-pub struct StateUpdateBuilder<F: RescueParameter>(PhantomData<F>);
-
-impl<F> StateUpdateBuilder<F>
+/// A function that takes as input:
+/// - a list of stake table entries (`Vec<(BLSVerKey, Amount, SchnorrVerKey)>`)
```

Comment:
Minor: mention that `Vec<BLSVerKey>` is not used here.

Reply:
I changed the input type. It's not taking `BLSVerKey` anymore.